require_build.php

```diff
@@ -44,4 +46,5 @@ class_alias('WoocommerceOnpay\OnPay\API\Util\Pagination', 'OnPay\API\Util\Pagina
     /**
      * Other Classes
      */
-    class_alias('WoocommerceOnpay\Alcohol\ISO4217', 'Alcohol\ISO4217');
\ No newline at end of file
+    class_alias('WoocommerceOnpay\Alcohol\ISO4217', 'Alcohol\ISO4217');
+    
```

Comment:
Something is funky right here ?

Reply:
unnecessary newline at end of file has been removed.